.travis.yml

```diff
@@ -5,7 +5,7 @@ node_js: stable
 env:
   - PATH=$HOME/purescript:$PATH
 install:
-  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
+  # - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
```

Comment:
Is the tag defined elsewhere? Won’t this make the download URL invalid otherwise?

Reply:
```suggestion
  # - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
  - TAG=v0.14.0-rc3
```

Reply:
Yea, we need to include a new line with the tag.

Reply:
Weird... Guess my script didn't properly account for this situation... I've pushed a commit to fix it.